docs/en/engines/table-engines/mergetree-family/aggregatingmergetree.md

```diff
@@ -35,6 +35,16 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 
 For a description of request parameters, see [request description](../../../sql-reference/statements/create/table.md).
 
+### Parameters of AggregatingMergeTree
+
+#### default_aggregate_function
+
+`default_aggregate_function` - a name of a [SimpleAggregateFunction](../../../sql-reference/data-types/simpleaggregatefunction.md) to be applied to all columns whose type is different from [AggregateFunction](../../../sql-reference/data-types/aggregatefunction.md) or [SimpleAggregateFunction](../../../sql-reference/data-types/simpleaggregatefunction.md). Optional parameter.
+The function is not applied to columns that are in primary key.
+
+If `default_aggregate_function` is not specified, no aggregate function is applied by default.
```

Comment:
It's actually, does mean that `any` aggregate function is applied

Reply:
Updated